kqueen/models.py

```diff
@@ -450,4 +450,4 @@ def namespace(self):
         Returns:
             str: Namespace (from organization)
         """
-        return self.organization.namespace
+        return self.organization.namespace if self.organization else None
```

Comment:
Maybe we remove this method? it contains ~10 lines, but useful part is ```self.organization.namespace```

Reply:
It is a property so that all objects will are consistent and have namespace attribute